.eslintrc.js

```diff
@@ -2,13 +2,18 @@
 
 require('@uniswap/eslint-config/load')
 
+const rulesDirPlugin = require('eslint-plugin-rulesdir')
+rulesDirPlugin.RULES_DIR = 'eslint_rules'
```

Comment:
we don't use underscores anywhere - this should be `eslint-rules` for consistency

Reply:
discussed in dms that we have `node_modules` already at the top level as an example